src/stratis_cli/_actions/_list_pool.py

```diff
@@ -44,6 +50,24 @@
 )
 
 
+def _metadata_version(mopool) -> Optional[MetadataVersion]:
+    try:
+        return MetadataVersion(int(mopool.MetadataVersion()))
+    except ValueError:  # pragma: no cover
+        return None
+
+
+def _volume_key_loaded(mopool) -> Union[tuple[bool, bool], tuple[bool, str]]:
+    """
+    The string result is an error message indicating that the volume key
+    state is unknown.
+    """
+    result = mopool.VolumeKeyLoaded()
+    if isinstance(result, int):
```

Comment:
Does the type get unmarshalled as an integer? I would expect it to be of type `dbus.Bool` based on my interactions with the Python wrapper in other contexts.